mobile_back_samsung/samsung/lib/sbe_config.hpp

```diff
@@ -33,6 +33,73 @@ common_setting {
     value: "4"
     name: "4 threads"
   }
+  acceptable_value {
+    value: "1"
+    name: "Single thread"
+  }
+  acceptable_value {
+    value: "2"
+    name: "2 threads"
+  }
+  acceptable_value {
+    value: "4"
+    name: "4 threads"
+  }
+  acceptable_value {
+    value: "8"
+    name: "8 threads"
+  }
+  acceptable_value {
+    value: "16"
+    name: "16 threads"
+  }
+}
+
+common_setting {
+  id: "configuration"
+  name: "configuration"
```

Comment:
`common_setting` are not used in UI.
It doesn't make sense to have `acceptable_value` for them because only the default value is used.
While `num_threads` is used in tflite-based backend, settings with IDs `configuration`, `share_results`, `cooldown` are not used anywhere in the project, they are dead weight. For Samsung backend `num_threads` is also likely a dead weight.
Recently we removed these settings to make backend configurations smaller and easier to read.
Please, revert changes that add `common_setting` values.

Reply:
Done